src/main/java/com/atherys/towns/facade/TownFacade.java

```diff
@@ -529,6 +527,14 @@ private void joinTownMessage(Player player, Town town) {
         townsMsg.info(player, "You have joined the town of ", townText);
     }
 
+    private boolean isLeaderOfPlayerTown(Player player) {
+        Resident resident = residentService.getOrCreate(player);
+        if (resident.getTown() != null) {
+            return resident == resident.getTown().getLeader();
```

Comment:
Can you change this to be `.equals`? I know the instances should always be the same, but it might change in the future.

Reply:
This has been resolved.